doc/source/developer-guides/general.rst

```diff
@@ -21,24 +26,35 @@ methods. In particular the developer should:
 
     platform = Platform("<platform_name>", "<path_to_runcard>")
 
-where ``<platform_name>`` is the name of the in the yaml file and
-``<path_to_runcard>`` is the path to the yaml file.
+where ``<platform_name>`` is the name of the yaml file and
+``<path_to_runcard>`` is its path.
+If the name of the runcard is ``<platform_name>.yml``, it is not necessary to also provide the ``<path_to_runcard>`` as input.
 
-Here you have an example for the structure of the yaml file:
+Here is an example of a yaml file with a brief explanation:
 
 .. code-block:: yaml
 
     name: icarusq
     nqubits: 1
     description: Controls the Tektronix AWG5204, AlazarTech ATS9371, ...
 
+At the beginning there is the name of the platform, the maximum number of supported qubits (``nqubits``)
+and a brief description.
+
+.. code-block:: yaml
+
     settings:
         delay_before_readout: 0
         delay_between_pulses: 0
         resonator_spectroscopy_max_ro_voltage: 5726.62338856
         rabi_oscillations_pi_pulse_min_voltage: 5291.34802850
         experiment_start_instrument: awg
 
+All the information useful to connect and set up the instruments are collected
```

Comment:
I think that `settings` contains also general information about the platform including the topology and not only information related to the instruments. I would suggest to modify this sentence.